<?xml version="1.0" ?>

<launch>
  <node pkg="urban_nav" type="gps_sim_nhabben" name="gps_sim_nhabben" />
</launch>